storybook/cypress/e2e/accordion.cy.ts

```diff
@@ -191,4 +191,62 @@ describe("Accordion", () => {
       .find(".dso-label.dso-label-warning")
       .should("exist");
   });
+
+  it("should render renvooi in the handle correctly", () => {
+    cy.visit("http://localhost:45000/iframe.html?id=core-accordion--renvooi");
+
+    cy.get("dso-accordion")
```

Comment:
Dit is nou zo'n cypress file waarin het gebruikt van `.hydrated` niet volledig is doorgevoerd. Zou nu wel een mooie gelegenheid zijn.

Reply:
Gedaan